persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/RowDecoratorImpl.java

```diff
@@ -43,4 +50,16 @@ public RowDecoratorImpl(TableName tableName, TableMetadata tableMetadata) {
     DecoratedKey decoratedKey = metadata.partitioner.decorateKey(key.serializeAsPartitionKey());
     return new ComparableKey<>(PartitionPosition.class, decoratedKey);
   }
+
+  @Override
+  public ByteBuffer getComparableBytes(Row row) {
+    Clustering key = metadata.partitionKeyAsClusteringComparator().make(primaryKeyValues(row));
+    DecoratedKey decoratedKey = metadata.partitioner.decorateKey(key.serializeAsPartitionKey());
+    ByteSource src = decoratedKey.asComparableBytes(ByteComparable.Version.DSE68);
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    for (int i = src.next(); i != END_OF_STREAM; i = src.next()) {
+      baos.write(i);
+    }
```

Comment:
should we simply use: `com.datastax.bdp.db.tries.util.ByteSourceUtil#readBytes(org.apache.cassandra.utils.ByteSource)`

Reply:
ah I didn't see that this existed - I will use it :+1:

Reply:
Sounds good, although which package is that from? (bdp means... DSE?)